### ISSUE TITLE
Do not allow deleting current client from GUI

### ISSUE THREAD

**Issue by searinminecraft** [gotify/server#677]
**Is your feature request related to a problem? Please describe.**

You could accidentally delete your current client from within the GUI.

**Describe the solution you'd like**

There should be an indication that its your current client, and delete/hide the delete button.

**Describe alternatives you've considered**

None

**Additional context**

Concept:

![image](https://github.com/user-attachments/assets/357a44c7-75bd-4416-93db-2fe153e278ac)

**Comment by jmattheis** [comment]
I don't think we have to disable the delete button but highlighting the current session seems reasonable.

**Comment by SulimanAbdulrazzaq** [comment]
I'd like to implement the maintainer-approved scope by marking the client whose ID matches the authenticated user's current client ID as the current session in the clients table. I'll add a focused UI regression assertion for the marker and preserve the existing delete behavior, as requested in the discussion.
